src/i18n/langDE.ts

```diff
@@ -134,9 +134,9 @@ export default {
             withName: 'die Daten für {name} sind {value}',
             withoutName: '{value}',
             separator: {
-                middle: ',',
```

Comment:
Thanks for your contribution! Can this solve your issue? This separator is just for splitting the sentences of `aria-label`. It won't be applied to the number format.

Reply:
Ah sorry, it won't help me then, it still is a problem and point of confusion for German users. Any idea where the number separator is defined?

Reply:
I'm afraid there is no way to define the number separator for now. This should be a new feature. For better i18n support, I think we can try to add it later.